.travis.yml

```diff
@@ -15,3 +15,8 @@ script:
   # our source code is currently in a subdirectory
   - cd cloud.foundry.cli
   - ./gradlew build
+  
+# send testreport to coveralls
+after_success:
+- ./gradlew jacocoTestReport coveralls
```

Comment:
Why are you running the `jacocoTestReport` test here, too? Isn't it part of `test`, which is run as part of `build` anyway?

I presume it only aggregates the results into the `.xml`, and running it twice won't affect the runtime of the builds very much?

Reply:
This pushes the results to coveralls.io